<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>936</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="buttonNew">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>New</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSort">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDelete">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonLoad">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>10</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonFind">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Find</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSave">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>10</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableContacts">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>911</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxSortBy">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>name</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>surname</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>patronymic</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>address</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>date</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>email</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>number</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEditFind">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonShowAll">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Show all</string>
    </property>
   </widget>
   <zorder>tableContacts</zorder>
   <zorder>buttonNew</zorder>
   <zorder>buttonSort</zorder>
   <zorder>buttonEdit</zorder>
   <zorder>buttonDelete</zorder>
   <zorder>buttonLoad</zorder>
   <zorder>buttonFind</zorder>
   <zorder>buttonSave</zorder>
   <zorder>comboBoxSortBy</zorder>
   <zorder>lineEditFind</zorder>
   <zorder>buttonShowAll</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>936</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPhoneBook_by_Yulia_Karma">
    <property name="title">
     <string>PhoneBook by Yulia Karma</string>
    </property>
   </widget>
   <addaction name="menuPhoneBook_by_Yulia_Karma"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
